### PR TITLE
Log overviews usage

### PR DESCRIPTION
**Pull request by enekid**
Comes from https://github.com/CartoDB/cartodb-management/issues/5201

### CODE REVIEW

**Comment by enekid** [comment]
The dataviews part is already completed.

File: lib/cartodb/models/dataview/base.js [hunk 1]
```diff
@@ -34,8 +34,14 @@ module.exports = class BaseDataview {
                 result = this.format(result, override);
                 result.type = this.getType();
 
-                return callback(null, result);
+                //Overviews logging
+                const stats = {};
 
+                if (flags && flags.usesOverviews) {
+                    stats.usesOverviews = true;
```
Comment (dgaubert) [inline lib/cartodb/models/dataview/base.js]: Shouldn't it be initialized to `false`?

Comment (enekid) [reply lib/cartodb/models/dataview/base.js]: `flags.usesOverviews` is only set to `true` if overviews are used. If it is `false` or it's `undefined` then `stats.usesOverviews` is not set.

In other words: We log only `stats.usesOverviews` if it is `true`

**Comment by enekid** [comment]
The part of logging the map instantiations that use overviews is still missing. WIP.

File: lib/cartodb/models/dataview/base.js (hunk 1 above)
Comment (dgaubert) [reply lib/cartodb/models/dataview/base.js]: Ok, got it. @enekid explain it to me off-line

Comment (oleurud) [reply lib/cartodb/models/dataview/base.js]: It will be easy to use Kibana if you log the `false` too

**Pull request merged**